### PR TITLE
HW2 Task04

### PR DESCRIPTION
**Pull request by LinaKrevetochka**

### CODE REVIEW

File: Homework2/Task04/task04.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+assert val_1 is val_2
+"""
+from collections import Callable
+from typing import Any
+
+
+def cache(func: Callable) -> Callable:
+    my_cache = {}
+
+    def wrapper(*args: Any) -> Any:
```
Comment (Cjkjvfnby) [inline Homework2/Task04/task04.py:21]: Does it cover all possible arguments?

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]
please add a test for 
```
def foo(nums: List[int]) -> List[int]:
   return [x // 2 for x in nums]
```

File: Homework2/Task04/task04.py (hunk 1 above)
Comment (LinaKrevetochka) [reply Homework2/Task04/task04.py:21]: I understood. 
I rewrote it

Comment (Cjkjvfnby) [reply Homework2/Task04/task04.py:21]: Actually this question was about keyword arguments.

**Review by Cjkjvfnby — APPROVED** [review]

**Pull request merged**